core-services/prow/03_deployment/deck-redirect.yaml

```diff
@@ -7,7 +7,7 @@ data:
   "default.conf": |
     server {
       listen 8080;
-      return 302 $scheme://deck-ci.apps.ci.l2s4.p1.openshiftapps.com$request_uri;
+      return 302 $scheme://prow.ci.openshift.org/$request_uri;
```

Comment:
NIT: We could leave this alone. We have a JIRA card for this and other redirections.

Reply:
Any downside to keeping it?